nodes/viz/texture_viewer.py

```diff
@@ -165,6 +161,10 @@ def __init__(self):
         items=theme_mode_options, default="default", update=updateNode
     )
 
+    def get_buffer(self,texture):
+        self.texture = texture
+        return self.texture
+
```

Comment:
i get it a image_name.png in /tmp folder, it is completely black but it is a progress!